make-iso.sh

```diff
@@ -19,10 +19,15 @@ then
     echo "Please run this script as root (sudo $@$0)."
     exit
 fi
+# /var/lmc cannot exist before building
+if [ -d /var/lmc ]
```

Comment:
Maybe think of different location for the build to occur, such as `/var/lmc/foo` instead of what's currently there. It may not be a good idea to delete that directory for _other_ developers who wish to use this file to build a personal ISO